HISTORY.md

```diff
@@ -24,6 +28,9 @@
 * User can set coordinate buffers separately for write queries.
 * Added option to enable duplicate coordinates for sparse arrays [#1504](https://github.com/TileDB-Inc/TileDB/pull/1504) 
 * Added support for writing at a timestamp by allowing opening an array at a timestamp (previously disabled).
+* Added special files with the same name as a fragment directory and an added suffix ".ok", to indicate a committed fragment. This improved the performance of opening an array on object stores significantly, as it avoids an extra REST request per fragment.
+* Added functionality to consolidation, which allows consolidating the fragment metadata footers in a single file by toggling a new config parameter. This leads to a huge performance boost when opening an array, as it avoids fetching a separate footer per fragment from storage.
+* Various reader parallelizations that boosted read performance significantly.
```

Comment:
add description of `.vac` files?

Reply:
Yeah, we will describe them in detail in the docs (and the format spec).